BSP/Simulator_V2/Src/BSP_SPI.c

```diff
@@ -0,0 +1,203 @@
+/* Copyright (c) 2022 UT Longhorn Racing Solar */
+
```

Comment:
Does this file need to be rewritten (so needs TODO's)?

Reply:
Yeah, this one is just a copy-paste from simulator v1 and needs to be updated